spec/models/spotlight/reindexing_log_entry_spec.rb

```diff
@@ -0,0 +1,49 @@
+describe Spotlight::ReindexingLogEntry, type: :model do
+  subject { FactoryGirl.build(:reindexing_log_entry) }
+
+  describe 'scope' do
+    before(:all) do
+      # we only want to persist these rows for the duration a given test run...
+      DatabaseCleaner.start
```

Comment:
Meh, doesn't bother me too much.  If you found it was necessary then it's 🆗 by me.